src/Octoshift/GithubApi.cs

```diff
@@ -459,5 +460,145 @@ public virtual async Task<string> GetArchiveMigrationUrl(string org, int migrati
+            };
+
+            var response = await _client.PostAsync(url, payload);
+            var data = JObject.Parse(response);
+
+            return data["data"]["user"].Any() ? (string)data["data"]["user"]["id"] : null;
+        }
+
+        public virtual async Task<bool> ReclaimMannequin(string orgId, string mannequinId, string targetUserId)
+        {
+            var url = $"{_apiUrl}/graphql";
+
+            var payload = BuildReclaimMannequinPayload(orgId, mannequinId, targetUserId);
+
+            var response = await _client.PostAsync(url, payload);
+            var data = JObject.Parse(response);
+
+            return data["data"]["createAttributionInvitation"].Any()
+                && (string)data["data"]["createAttributionInvitation"]["source"]["id"] == mannequinId
+                && (string)data["data"]["createAttributionInvitation"]["target"]["id"] == targetUserId;
+        }
+
+        private object BuildListMannequinPayload(string orgId, string cursor)
+        {
+            return new
+            {
+                query = @"query ($id: ID!, $after: String) {
+	                node(id: $id) {
+		                ... on Organization {
+			                mannequins(first: 100, after: $after) {
+				                pageInfo {
+					                endCursor
+					                hasNextPage
+				                }
+				                totalCount
+				                nodes {
+					                login
+					                id
+					                claimant {
+						                login
+						                id
+					                }
+				                }
+			                }
+		                }
+	                }
+                }
+                ",
+                variables = new { after = cursor, id = orgId }
+            };
+        }
+
+        private object BuildReclaimMannequinPayload(string orgId, string mannequinId, string targetUserId)
+        {
+            return new
+            {
+                query = @"mutation(
+                      $orgId: ID!,
+                      $sourceId: ID!,
+                      $targetId: ID!,
+                    ){
+                      createAttributionInvitation( input: {
+                        ownerId: $orgId,
+                        sourceId: $sourceId,
+                        targetId: $targetId
+                      })
+                      {
+                        source {
+                          ... on Mannequin {
+                            id
+                            login
+                          }
+                        }
+
+                        target {
+                          ... on User {
+                            id
+                            login
+                          }
+                        }
+                      }
+                    }
+                ",
+                variables = new { orgId, sourceId = mannequinId, targetId = targetUserId }
+            };
+        }
```

Comment:
same

Reply:
refactored to inline query in [d48c3](https://github.com/github/gh-gei/pull/271/commits/d48c3bfd20422b7a65ac851b6232341c29f88a3e)